src/core/Akka/Actor/ActorCell.Children.cs

```diff
@@ -199,7 +199,7 @@ protected void UnreserveChild(string name)
         /// </summary>
         /// <param name="actor">TBD</param>
         /// <returns>TBD</returns>
-        public ChildRestartStats InitChild(IInternalActorRef actor)
+        public ChildRestartStats? InitChild(IInternalActorRef actor)
```

Comment:
`ChildRestartStats` will be `null` if we don't have a child by this name - should _never_ happen but it has always been theoretically possible